cfme/tests/cli/test_appliance_console.py

```diff
@@ -1406,8 +1419,51 @@ def test_appliance_console_static_dns():
         caseimportance: low
         caseposneg: negative
         initialEstimate: 1/6h
+        testSteps:
+            1. configure static ipv4
+            2. stop and start EVM service
+            3. reconfigure static ipv4
+            4. configure static ipv6
+            5. stop and start EVM service
+            6. reconfigure static ipv6
+        expectedResults:
+            1.
+            2.  check evm service status while stop and start
+            3. check default secondary DNS is listed or not while entering  DNS
+            4.
+            5. check evm service status while stop and start
+            6. check default secondary DNS is listed or not while entering  DNS
     """
-    pass
+    appliance = temp_appliance_preconfig_long
+    command_set = ("ap", RETURN, app_con_menu["config_net"], ip_version_menu, RETURN, RETURN,
+                   RETURN, RETURN, secondary_dns, RETURN, TimedCommand("Y", 120))
+    appliance.appliance_console.run_commands(command_set, timeout=30)
+
+    def _stop_start_evmserverd(appliance):
+        command_set = ("ap", RETURN, app_con_menu["stop_evm"], "Y")
+        appliance.appliance_console.run_commands(command_set, timeout=30)
```

Comment:
**Question** Is there a reason you are starting and stopping via the appliance console and not just running:

``` appliance.evmserverd.stop()
    appliance.evmserverd.start()
    appliance.wait_for_web_ui()```

Reply:
appliance console recommends to stop and start appliance after network configuration so performing stop and start operation and test is related to console so trying to perform the most operation via console only.
Test is not related to UI so not thought about checking UI, I think service check is fine. What is your view?

Reply:
**Required** I think starting and stopping via the ap is fine but I would use the property `appliance.evmserverd.wait_for_running()`

Reply:
@jawatts added wait for function after stop and start the EVM process